<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playground</class>
 <widget class="QMainWindow" name="Playground">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playground</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="create_child_button_">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>32</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar crian√ßa</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="has_ball_checkbox_">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tem bola?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="play_interval_line_edit_">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="quiet_interval_line_edit_">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_text_edit_">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>761</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
